api/scripts/certification/generate-certification-results.js

```diff
@@ -47,16 +47,15 @@ async function main() {
     scores.forEach((score) => console.log(score));
   } catch (error) {
     console.error(error);
-    process.exit(1);
+    throw new Error(error);
   }
```

Comment:
En fait ici, on peut ne pas catcher l'erreur, ca sera recupéré dans le catch du main. Ca simplifie un peu

```suggestion
  }
```